UnityProject/Assets/Scripts/Antagonists/Objectives/Assassinate.cs

```diff
@@ -33,9 +33,10 @@ public override bool IsPossible(PlayerScript candidate)
 		/// </summary>
 		protected override void Setup()
 		{
-			// Get all ingame players except the one who owns this objective and players who have already been targeted
+			// Get all ingame players except the one who owns this objective and players who have already been targeted and the ones who cant be targeted
 			List<ConnectedPlayer> playerPool = PlayerList.Instance.InGamePlayers.Where( p =>
-				(p.Script != Owner.body) && !AntagManager.Instance.TargetedPlayers.Contains(p.Script)
+				(p.Script != Owner.body) && !AntagManager.Instance.TargetedPlayers.Contains(p.Script) && p.Script.mind.occupation != null && p.Script.mind.occupation.IsTargeteable

```

Comment:
p.Script.mind.occupation.IsTargeteable is not correct needs to be p.Script.mind.occupation.CanTarget